package.json

```diff
@@ -40,7 +40,7 @@
     "@mui/system": "5.6.2",
     "@solana/web3.js": "1.39.1",
     "@types/masknet__global-types": "workspace:*",
-    "@types/react": "18.0.6",
+    "@types/react": "18.0.2",
```

Comment:
Why you're downgrading this lib?

Reply:
let me check why it happened.

Reply:
When i pull the commit the package.json is not updating in my fork, what could be the issue?

Reply:
- Fixed, and synced with latest development commit.